_posts/2023-04-29-release.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Release 2023.17
+author: Priya
+date: 2023-04-29
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### Email Notification for Pipeline Errors.(an alpha feature on demand)
+* Mammoth have added an email notification feature to alert users when a task or action fails during pipeline execution.
+
+
+### Defect fixes
+
+* Fixed an issue where zip file uploads were not working.
+* Fixed an issue where users were unable to move a DS/Folder to the Data Library.
+* Fixed an issue where column renames in foreign view did not get reflected in the join rule's pipeline description until the page was refreshed.
+* Fixed an issue where in/out links did not disappear when the dependent dataview was deleted.
+* Fixed an issue where dragging a file directly from the browser to the data library did not work every time.
+* Fixed an issue where runtime errors were not being displayed for internal export task failures.
```

Comment:
Fixed an issue for missing display of internal export task errors.

Reply:
Missing the two security tickets related updates?

Reply:
@prmammoth   Do we need to include those in external release notes?
MVP-9082 , MVP-9091

Reply:
I was thinking, yes because we are anyway required to report our status on penetration test fixes. I can work to tune the language

Reply:
Added security tickets.